### ISSUE TITLE
service-catalog does not pass the operation to last_operation

### ISSUE THREAD

**Issue by jmrodri** [kubernetes-incubator/service-catalog#732]
The OpenServiceBroker [spec](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#polling-last-operation) specifically states **Query-string field** vs **Request Field**:

##### Parameters #####

The request provides these query string parameters as useful hints for brokers.

|  Query-String Field | Type  | Description  |
|---|---|---|
| service_id  |  string | ID of the service from the catalog.  |
| plan_id  | string  | ID of the plan from the catalog.  |
| operation  |  string | A broker-provided identifier for the operation. When a value for <code>operation</code> is included with asynchronous responses for [Provision](#provisioning), [Update](#updating_service_instance), and [Deprovision](#deprovisioning) requests, the broker client should provide the same value using this query parameter as a URL-encoded string.  |

But if you look at the url string the operation is not appended to the url:

[`
pollingFormatString                    = "%s/v2/service_instances/%s/last_operation"
`](https://github.com/kubernetes-incubator/service-catalog/blob/master/pkg/brokerapi/openservicebroker/open_service_broker_client.go#L43)

We send in the operation in the request body for a `GET` call. But never append it to the url as a query parameter.

```
func (c *openServiceBrokerClient) pollBroker(ID string, operation string) error {
	pollReq := brokerapi.LastOperationRequest{}
	if operation != "" {
		pollReq.Operation = operation
	}

	pollingURL := fmt.Sprintf(pollingFormatString, c.url, ID)
	for i := 0; i < pollingAmountLimit; i++ {
		glog.V(3).Infof("Polling broker %v at %s attempt %v", c.name, pollingURL, i+1)
		pollResp, err := sendOSBRequest(c, http.MethodGet, pollingURL, pollReq)
```
[Lines 302-311](https://github.com/kubernetes-incubator/service-catalog/blob/master/pkg/brokerapi/openservicebroker/open_service_broker_client.go#L302-L311)

**Comment by MHBauer** [comment]
It's interesting the example doesn't show the query parameters at all. `service_id` and `plan_id` are explicitly mentioned as optional, but `operation` is not.

I can read that section as "if the broker provides it in a response, the client should echo it back on the next request".  None of that sounds required to me.

I'd say this is more of a bug in the spec definition than our implementation until the spec is clarified.

**Comment by pmorie** [comment]
@vaikas-google can you handle this in #717 ?

**Comment by jmrodri** [comment]
@MHBauer that's part of the problem. It *is* being returned by the broker but the catalog is not echoing it back as a query parameter.

**Comment by vaikas-google** [comment]
Actually properly addressed by a follow up in #757
